File: followingApp/followingApp/followingApp/app/src/main/java/com/example/idolmastercalendar/IdolData.java
package com.example.idolmastercalendar;

import androidx.annotation.DrawableRes;

public class IdolData {

    String idolName;
    @DrawableRes int idolImage;

    public IdolData(String idolName, @DrawableRes int idolImage) {
        this.idolName = idolName;
        this.idolImage = idolImage;
    }

    public String getIdolName() {
        return idolName;
    }

    public int getIdolImage() {
        return idolImage;
    }
}

File: WimiProject/WimiProject/app/src/main/java/com/wimi/idolmaster/domain/model/IdolData.kt
package com.wimi.idolmaster.domain.model

import androidx.annotation.DrawableRes
import androidx.databinding.ObservableField

data class IdolData(
    var idolName: String,
    @DrawableRes var idolImage: Int
) {
    val isSelected = ObservableField(false)
}